backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -41,4 +41,11 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
 
+.Additional resources
+
+For more information on installing OADP to back up and restore virtual machines, see:
```

Comment:
Please fix all of the unaddressed comments on this PR before resubmitting for merge review.